ring/util.go

```diff
@@ -96,6 +96,19 @@ func WaitInstanceState(ctx context.Context, r ReadRing, instanceID string, state
 // WaitRingStability monitors the ring topology for the provided operation and waits until it
 // keeps stable for at least minStability.
 func WaitRingStability(ctx context.Context, r *Ring, op Operation, minStability, maxWaiting time.Duration) error {
+	return waitStability(ctx, r, op, minStability, maxWaiting, HasReplicationSetChanged)
+}
+
+// WaitRingTokensStability waits for the Ring to be unchanged at
+// least for minStability time period, excluding transitioninig between
```

Comment:
[nit]
```suggestion
// least for minStability time period, excluding transitioning between
```

Reply:
thank you, I didn't realize that VS code has no spell check by default like Rider. Now installed.